playwright.config.ts

```diff
@@ -70,7 +70,7 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: "npm run dev",
+    command: process.env.CI ? "npm start" : "npm run dev",
```

Comment:
`npm run dev` is fine for local development, but for CI, we want a production build which runs much more faster than a dev build.